apps/vs-code-designer/src/app/utils/codeless/startDesignTimeApi.ts

```diff
@@ -190,12 +190,13 @@ export function startDesignTimeProcess(
 }
 
 export function stopDesignTimeApi(): void {
+  ext.outputChannel.appendLog('Stopping Design Time Api');
   if (ext.designChildProcess === null || ext.designChildProcess === undefined) {
     return;
   }
-
+  cp.exec('echo Stop DesignTime Api');
```

Comment:
Does this work for linux and windows?

Reply:
ah, I need to remove this. This was for debugging purposes. Changed it to ext.outputChannel